tests/test_table_row_classification.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import pandas as pd
```

Comment:
We might want to add a separate set of dependencies for the test suite. Currently, pandas doesn't need to be a dependency for usage of the library (it's used in type annotation, but we can hide that).

Reply:
I've just added pandas to the github actions workflow for now. We can add a test optional dependencies in a follow up PR if desired.